modules/pytket-quantinuum/pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -407,6 +409,14 @@ def process_circuits(
                 "utf-8"
             )
 
+        try:
+            pytket_pass = cast(BasePass, kwargs.get("pytketpass"))
+        except ValueError:
+            pytket_pass = None
+
+        if pytket_pass is not None:
+            basebody["options"]["tket-pass"] = pytket_pass.to_dict()
```

Comment:
For future extensions I think we should have a `tket` field under `options`, with further subfields, of which `compilation-pass` can be one. So:
```
options:
    tket:
        compilation-pass: ...
        ...
    ...
```
instead of
```
options:
    tket-pass: ...
    ...
```